### PR TITLE
style AcceptJobScreen screen buttons

### PR DESCRIPTION
**Pull request by ikemanep1**
1: threw a scrollview in the previous component (job details screen)
2: Restyled buttons to be touchableOpacity

Simulator Screen Shot - iPhone SE (2nd generation) - 2020-06-08 at 16 27 45

### CODE REVIEW

**Review by Jaywinebrenner — DISMISSED** [review]
Nicccce. Lookin' good.

File: screens/pilot/JobDetailsScreen.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ function JobDetailsScreen(props, { getClientProfiles }) {
   }
 
   return (
+    <ScrollView>
```
Comment (LeePedersen) [inline screens/pilot/JobDetailsScreen.js]: ScrollView can be used instead of View to make the code DRYer

File: screens/pilot/AcceptJobScreen.js [hunk 2]
```diff
@@ -22,8 +22,14 @@ function AcceptJobScreen(props, { acceptJob }) {
   return (
     <View style={styles.container}>
       <Text style={styles.servicesText}>Are you sure?</Text>
-      <Button title="Yes" onPress={accept} />
```
Comment (LeePedersen) [inline screens/pilot/AcceptJobScreen.js:25]: Make sure you're paying attention to indentation

**Review by LeePedersen — DISMISSED** [review]
Good job

**Review by GACOSTA1988 — APPROVED** [review]

**Review by LeePedersen — APPROVED** [review]
Thanks!

**Pull request merged**